troubleshooting.rst

```diff
@@ -22,9 +22,25 @@ This is usually due to an archive metadata bit used by some OS's that Horizon si
 
 ........
 
+Homebrew Launcher only launches album?
+--------------------------------------
+
+Make sure you hold the R button while launching Album or any game. Homebrew Launcher should load now.
+
+........
+
+I've injected the payload but there's a black screen after booting?
+-------------------------------------------------------------------
+
+Try again. 
+
+Remove everything from your SD card EXCEPT the Nintendo folder. Start the process of installing Atmosphere/Kosmos again.
+

```

Comment:
Mention somehow that you should try reformatting to FAT32. One of the big causes of this is people are trying to use a new exFAT SD card in Atmosphere without the exFAT update.

Eventually I'll add an actual page with steps on reformatting.